whitepapers/game-theory.md

```diff
@@ -0,0 +1,121 @@
+
+Player 1’s available strategies are the rows (Quiet or Confess) and their corresponding payoffs are the first numbers in each cell. 
+Players 2’s available strategies are the columns and their corresponding payoffs are the second numbers in the cells. 
+If player 1 stays Quiet and player 2 stays Quiet the game ends in the top left corner of the matrix. If both players Confess the game ends in the bottom right corner of the matrix and so on. 
+
+### Strategies
+To see which strategy each player will choose we should look at each move in isolation. From player 1’s perspective, what should he do if he thinks player 2 will stay Quiet?    
+![](../.gitbook/assets/matrix_2.png)
+
+We can see that player 1 should Confess because if he stays Quiet he will get one month in jail and we have already stated that both players prefer less time in jail. 
+What about if player 1 thought that player 2 was going to Confess?    
+![](../.gitbook/assets/matrix_3.png)
+
+Again it seems that player 1 should Confess as it leads to 8 months jail time rather than the 12 on offer if player 1 stays Quiet. 
+Putting this together we reach an important conclusion:  
+**Player 1 is better off confessing regardless of player 2’s strategy.**  
+
+Let’s look at player 2’s perspective, assuming he thinks player 1 will stay quiet:    
+![](../.gitbook/assets/matrix_4.png)
+
+Looking at the second numbers now we can see that like player 1, player 2 should confess as well: he will be set free instead of getting 1 month in jail.     
+![](../.gitbook/assets/matrix_5.png)
+
+Once again it looks like Confess should be the chosen strategy even if player 2 thinks player 1 will also Confess. 
+
+**Player 2 is also better off confessing regardless of player 1’s strategy.**
+
+### Assumptions and Conclusions
+- We assumed that both players' preferences were to minimise their jail time 
+- We assumed both players were self-interested (i.e they don't care about their friends’ fate)
+- We assumed only one interaction
+- We assumed the players could not interact and plan their responses in advance 
+
+These assumptions led to a sub-optimal outcome in the game (Confess, Confess). We can see that had both players stayed Quiet, they would have received less jail time. This is an **unstable equilibrium** however, because (as we saw) both players are motivated to opt for Confess if they believe the other will stay Quiet.   
+Confess, Confess is therefore the only Nash equilibrium. A Nash equilibrium is a state in a game where no player wishes to deviate from their strategy, given what the other players are doing.     
+**The only dominant strategy in the prisoner's dilemma is Confess, Confess. If both players were able to cooperate with each other and stay Quiet** **however, they would have achieved a better outcome.**   
+
+**This is an important conclusion as it shows us that two individuals may not cooperate, despite it appearing to be the best strategy for both.**
```

Comment:
```suggestion
These assumptions led to a sub-optimal outcome in the game (Confess, Confess). We can see that had both players stayed Quiet, they would have received less jail time. This is an **unstable equilibrium** however, because (as we saw) both players are motivated to opt for Confess if they believe the other will stay Quiet.

Confess, Confess is therefore the only Nash equilibrium. A Nash equilibrium is a state in a game where no player wishes to deviate from their strategy, given what the other players are doing.

**The only dominant strategy in the prisoner's dilemma is Confess, Confess.** If both players were able to cooperate with each other and stay Quiet however, they would have achieved a better outcome. This is an important conclusion as it shows us that **two individuals may not cooperate, despite it appearing to be the best strategy for both.**
```